test_anagram (__main__.TestExercice) ... ok
test_doubles (__main__.TestExercice) ... ok
test_frequence (__main__.TestExercice) ... ERROR
test_grades (__main__.TestExercice) ... ERROR
test_order (__main__.TestExercice) ... ok

======================================================================
ERROR: test_frequence (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 77, in test_frequence
    output = [exercice.frequence(s) for s in sentences]
  File "test_exercice.py", line 77, in <listcomp>
    output = [exercice.frequence(s) for s in sentences]
  File "/home/runner/work/2021h-ch6-1-exercices-phhad/2021h-ch6-1-exercices-phhad/exercice.py", line 47, in frequence
    sorted_keys = sorted(frequency, reverse=True, key=frequency.__getitem__())
TypeError: __getitem__() takes exactly one argument (0 given)

======================================================================
ERROR: test_grades (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 62, in test_grades
    output = exercice.best_grades(value)
  File "/home/runner/work/2021h-ch6-1-exercices-phhad/2021h-ch6-1-exercices-phhad/exercice.py", line 39, in best_grades
    return {student_grades}
TypeError: unhashable type: 'dict'

----------------------------------------------------------------------
Ran 5 tests in 0.001s

FAILED (errors=2)
